Makefile

```diff
@@ -14,8 +14,8 @@
 
 -include Makefile.overrides
 
-MAISTRA_VERSION        ?= 2.2.2
-MAISTRA_BRANCH         ?= maistra-2.2
+MAISTRA_VERSION        ?= 2.3.0
+MAISTRA_BRANCH         ?= maistra-2.3
 REPLACES_PRODUCT_CSV   ?= 2.2.1
 REPLACES_COMMUNITY_CSV ?= 2.2.1
```

Comment:
I think this needs to be updated, since we've released 2.2.2.

Reply:
Done